server/services/educationAndWorkPlanApiPersonalLearningPlanService.ts

```diff
@@ -0,0 +1,27 @@
+import { RestClientBuilder } from '../data'
+import { EducationAndWorkPlanApiClient } from '../data/interfaces/educationAndWorkPlanApiClient'
+import { PersonalLearningPlanActionPlan } from '../interfaces/personalLearningPlanGoals'
+import toPersonalLearningPlanActionPlan from '../interfaces/mappers/personalLearningPlanActionPlanMapper'
+import logger from '../../logger'
+import PersonalLearningPlanService from './personalLearningPlanService'
+
+/**
+ * Implementation of [PersonalLearningPlanService] that uses the [EducationAndWorkPlanApiClient] as data source
+ * for PLP related data.
+ */
+export default class EducationAndWorkPlanApiPersonalLearningPlanService extends PersonalLearningPlanService {
```

Comment:
The diff is showing this as a new class, but actually all I've done is move it out of `personalLearningPlanService.ts`